pkg/api/index.go

```diff
@@ -45,15 +45,17 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 		return nil, err
 	}
 
-	// Set locale to the preference, otherwise fall back to the accept language header.
-	// In practice, because the preference has configuration-backed default, the header
-	// shouldn't frequently be used
+	// Locale is used for some number and date/time formatting, whereas language is used just for
+	// translating words in the interface
```

Comment:
Language is the... language the words in the interfaces are translated into. 

Locale is how things other than words are localised, such as number and date formatting.

The only _language_ we support is en-US (we don't have `en-GB` at the moment), but I don't want to see dates formatted according to US (as mm/dd/yyyy) convention. Similarly, a native German might prefer the interface in English, but numbers formatted according to German conventions with commas for thousands separators.

Right now we're not actually implementing the distinction (we don't allow setting these values separately, nor do we actually consistently do locale-aware formatting of values), but I wanted to make sure we weren't saving preferences under names we might regret later. Right now we only use it for language, so that's what it's being called all throughout the code.

Reply:
Ah, makes sense! Thanks for explaining this 🙏🏻